container/slice/slice.go

```diff
@@ -387,3 +387,13 @@ func VerifyMaxLength[T any](v []T, max int) error {
 	}
 	return nil
 }
+
+// Index finds the index of an element in a slice.
```

Comment:
```suggestion
// Index finds the index of an element in a slice. Returns -1 if the element is not present in the slice.
```

Reply:
Actually, this is identical to [slices.Index](https://cs.opensource.google/go/go/+/refs/tags/go1.22.3:src/slices/slices.go;l=94)? Can you use that?